test/Constraints/diag_missing_arg.swift

```diff
@@ -66,6 +66,8 @@ param2FuncNonNamed3(1) // expected-error {{missing argument for parameter #2 in
 param2FuncNonNamed3("foo") // expected-error {{missing argument for parameter #2 in call}} {{26-26=, <#String#>}}
                            // FIXME: Bad diagnostic. Could this be #1?
 
+func unlabeledParamFollowingVariadic(_: Any..., _: Any, _: Any) {} // expected-warning {{a parameter following a variadic parameter requires a label}}
```

Comment:
(which matches @rintaro 's expected-warning/expected-error; the FIXME can be removed)